docs/Reference/Validation-and-Serialization.md

```diff
@@ -390,6 +390,12 @@ body, URL  parameters, headers, and query string. The default
 [ajv](https://ajv.js.org/) validation interface. Fastify uses it internally to
 speed the validation up.
 
+> **Note** 
+> Since `validatorCompiler` is loaded on the first schema validation it's possible
+> that its value is falsey. If you need to validate some schema at runtime
+> (in contexts not covered by Fastify by default)
+> take a look a [these helpers](https://fastify.dev/docs/latest/Reference/Request/#validateinputdata-schema--httpstatus-httpstatus)
```

Comment:
Isn't it the duplication of
https://fastify.dev/docs/latest/Reference/Server/#validatorcompiler ?

>This property can be used to get the schema validator. If not set, it will be null until the server starts, then it will be a function with the signature function ({ schema, method, url, httpPart }) that returns the input schema compiled to a function for validating data. The input schema can access all the shared schemas added with [.addSchema](https://fastify.dev/docs/latest/Reference/Server/#add-schema) function.

Reply:
I didn't notice that documentation, but it's partially correct since there
are times in which the validatorCompiler is falsy even if the server has
started successfully.
What's your opinion about this?

About the absolute path I can fix that.

Reply:
Possibly I can update that part of the documentation instead of adding the new note